tests/test_utils/perf/dags/elastic_dag.py

```diff
@@ -51,6 +51,24 @@ def parse_time_delta(time_str: str):
     return timedelta(**time_params)  # type: ignore
 
 
+def parse_start_date():
+    """
+    Returns the start date for the elastic DAGs and string to be used as part of their ids.
+
+    :return Tuple[datetime.datetime, str]: A tuple of datetime.datetime object to be used
+        as a start_date and a string that should be used as part of the dag_id.
+    """
+
+    if "PERF_START_DATE" in os.environ:
```

Comment:
PERF_START_DATE env var and PERF_START_AGO env variable  are mutually exclusive. Am I right?  If so, it makes sense to me to throw an exception to warn the user of the unsupported configuration.

Reply:
Currently, if `PERF_START_DATE` is defined, then the value of `PERF_START_AGO` (if defined as well) will simply be ignored. Are you sure we should throw an exception if both of them are defined? They are mutually exclusive, but having both of them defined does not prevent parsing the dag file.

Reply:
It will be ignored without any explicit message to the user, which may cause problems. I prefer to add one more check than then diagnose non-obvious behavior.

Reply:
I agree. Added raising exception.